packages/@sanity/core/src/actions/graphql/extractFromSanitySchema.js

```diff
@@ -375,7 +375,7 @@ function extractFromSanitySchema(sanitySchema, extractOptions = {}) {
   }
 
   function getDocumentInterfaceFields() {
-    const isNullable = typeof nonNullDocumentFields === 'boolean' ? !nonNullDocumentFields : true
```

Comment:
I'm guessing this is where the default to true condition was expected to occur.

This doesn't happen because the condition `typeof nonNullDocumentFields === 'boolean'` will always be true.

Reply:
Yeah this check is unnecessary given that the flag is explicitly a boolean now - good point.